scripts/for_combined_data.R

```diff
@@ -0,0 +1,17 @@
+# Load necessary library
+library(dplyr)
+
+# Define directory containing CSV files
+input_dir <- "/Users/kachanta/Desktop/prostate_Natalie/output_5"
```

Comment:
we can't use hard-coded paths in the script because people will not be able to use them. There should be a script or another reference file where you set these variables